packages/design-system/src/components/snackbar/snackbarContainer.js

```diff
@@ -125,13 +127,29 @@ export const SnackbarContainer = ({
     }
   }, [max, notifications, onRemove]);
 
+  // Announce messages to screen reader when a new message shows up
+  useEffect(() => {
+    notifications.forEach((notification) => {
+      if (!announcedNotifications.current.has(notification)) {
+        // speak the message
+        const message = `${notification.message} ${
+          notification.actionHelpText ? notification.actionHelpText : ''
```

Comment:
```suggestion
          notification.actionHelpText || ''
```

Reply:
😂 I should've done this in the first place, nice catch